common/motion_utils/include/motion_utils/trajectory/trajectory.hpp

```diff
@@ -607,8 +607,12 @@ double calcLateralOffset(
     return std::nan("");
   }
 
-  const auto p_front = tier4_autoware_utils::getPoint(overlap_removed_points.at(seg_idx));
-  const auto p_back = tier4_autoware_utils::getPoint(overlap_removed_points.at(seg_idx + 1));
+  const auto p_indices = overlap_removed_points.size() - 2;
```

Comment:
If the original size is 10 and the overlap_removed size is 9 and the seg_idx == 9, still this code crashes.
So it should be "p_indices > seg_idx"

Reply:
Done https://github.com/autowarefoundation/autoware.universe/pull/6653/commits/c263222b19b7efb4033cf8d39ecbb3ab91f80d50